src/core/function_assets_cdn.tf

```diff
@@ -66,7 +66,7 @@ module "function_assets_cdn" {
 
   os_type                                  = "linux"
   runtime_version                          = "~4"
-  linux_fx_version                         = "NODE|14"
+  linux_fx_version                         = "NODE|18"
```

Comment:
I think we need to update function_assets_cdn_staging_slot (line 104) before updating the main slot.

Reply:
Yes, @Garma00 we should do what @fabriziopapi said. We are going to release on staging first.

Reply:
Thanks, this was an old pr and I forgot to change the staging slot, did in [32f5fde](https://github.com/pagopa/io-infra/pull/415/commits/32f5fde42bdf2c10ba00d194a0190dd5dcdcfb80)